pandas/tests/io/parser/test_parsers.py

```diff
@@ -99,3 +102,14 @@ def read_table(self, *args, **kwds):
         kwds = kwds.copy()
         kwds['engine'] = self.engine
         return read_table(*args, **kwds)
+
+
+class TestParameterValidation(object):
```

Comment:
> so okay that this will get run once for every engine, even though it's the same test?

Actually, better idea: move it to `test_common.py` (the directory above)